frontend/src/components/Signup/Signup.js

```diff
@@ -31,6 +31,7 @@ const Signup = () => {
     const error = {}
     const emailRegex = /^[^\s+@]+@[^\s@]+\.[^\s@]{2,}$/i
     const phoneRegex = /^\+?\d{1,3}[- ]?\d{3}[- ]?\d{3}[- ]?\d{4}$/
+    values.email = values.email.toLowerCase()
```

Comment:
What if the account is created using the API directly(i.e. without frontend)? Does your code cover that case?

Reply:
Updated changes for this case in backend

Reply:
it will be saved as aditi@gmail.com